BridgeCareApp/BridgeCare/Models/AttributeByYearModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using BridgeCare.Interfaces;
+using System;
+using System.Data;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace BridgeCare.Models
+{
+    public class AttributeByYearModel
+    {
+        public AttributeByYearModel()
+        {
+            yearlyvalues = new List<AttributeYearlyValueModel>();
```

Comment:
yearlyValues

Reply:
actually that is a member property/variable, going by earlier comment changed to PascalCase